python/lsst/afw/table/aggregates/aggregatesContinued.py

```diff
@@ -40,6 +40,11 @@ class PointKey(with_metaclass(TemplateMeta, object)):
 PointKey.register((np.float64, 2), aggregates.Point2DKey)
 
 
+# Because Boxes aren't templates themselves, we don't expose the fact that
```

Comment:
Should this be in a docstring?

Reply:
I don't think so; the potential confusion this comment is trying to avoid is from seeing a change in the pattern established in the rest of the file, and hence I don't think there's much to explain for someone who isn't reading the code.